server/testutils/jest.setup.ts

```diff
@@ -26,6 +26,7 @@ const apiSpecs = {
     command: (openAPIUrl: string) => `if [ ! -f ${apiSpecPaths.cas1Spec} ]; then
     curl -s "${openAPIUrl}" |
     sed -E 's@/premises@/cas1/premises@g' |
+    sed -E 's@ /out-of-service-beds@ /cas1/out-of-service-beds@g' |
```

Comment:
Not a big deal but I would add this change to the commit where we add the new client method

Reply:
ok yeah, have done